src/charmory/export.py

```diff
@@ -161,7 +161,7 @@ def draw_boxes_on_image(
             width=ground_truth_width,
         )
 
-    if pred_boxes is not None and len(pred_boxes) > 1:
+    if pred_boxes is not None and len(pred_boxes) > 0:
```

Comment:
Why did this get changed out of curiosity? I thought that prediction 0 in object detection was the background?

Reply:
hmm, maybe that's the case for some models but not all? I can't say I've noticed whether or not the very first prediction in the list is or isn't the background...

but either way, I'd assume the score for that background "detection" would be 0/near 0 and will have been filtered out before this condition gets hit. I think the original `> 1` check was just a bug when it was first written (my fault) and not noticed because all our previous examples were multi-object detections anyway. with the license plate example I was using here, there's only one expected detection per image, so every export was failing on this (bad) conditional.

Reply:
Maybe I am incorrect I'm having trouble finding that out, but I thought I saw that somewhere. But I guess it's not the case for most object detection models then. And yea that is probably true, so checking for the score to be a higher number should work.